pandas/core/generic.py

```diff
@@ -2057,54 +2057,12 @@ def empty(self) -> bool_t:
     def __array__(self, dtype: npt.DTypeLike | None = None) -> np.ndarray:
         return np.asarray(self._values, dtype=dtype)
 
-    def __array_wrap__(
```

Comment:
What's the reason this was removed? 

(dask has failures that seem to be caused by this, eg https://github.com/dask/dask/runs/4823351165?check_suite_focus=true)

Reply:
i was under the impression this was superceded by array_ufunc.  If not, feel free to reinstate (with test)